test/spec/oidc/verifyToken.ts

```diff
@@ -1,3 +1,50 @@
+const modulesToMock = {
+  validateClaims: '../../../lib/oidc/util/validateClaims',
```

Comment:
nit: [jest moduleNameMapper](https://jestjs.io/docs/en/configuration#modulenamemapper-objectstring-string--arraystring) probably can save some dots here.

The config can be something like:
```
"moduleNameMapper": {
    "@lib/(.*)$": "<rootDir>/lib/$1"
}
```